src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -113,9 +113,26 @@ const getSignOutLink = (settings) => {
   }
 };
 
+const getDefaultBackSignInLink = (settings, appState) => {
+  const messagesObjs = appState?.get('messages');
+  if (messagesObjs && Array.isArray(messagesObjs.value)
+    && messagesObjs.value.some(messagesObj => messagesObj.i18n?.key === 'idx.return.link.expired')) {
+    return [
+      {
+        'type': 'link',
+        'label': loc('goback', 'login'),
```

Comment:
should we use `backToSignin` instead of `goback` ? Because no need to have `Sign In` in uppecases